### DISCUSSION TITLE
Can I get Profiles/Settings Back?

### DISCUSSION THREAD

**Discussion by R0b0ty** [borgbase/vorta#2069] [FAQ]
I deleted my distro's version of Vorta (`sudo dnf remove vorta`) in favor of the Flatpak version, as it has the latest version.  When I launch the Flatpak version, none of my previous profiles and repo information were retained.  I didn't expect the removal of the distro Vorta app to affect my settings.  Is there a way to get this back?  I did a search for the `settings.db` file, but could not find it.  Thanks!

**Comment by m3nu** [comment]
Have you looked in `.config/vorta`? I doubt the package will remove files in your home folder. `find` may not search hidden folders.

**Reply by R0b0ty** [reply]
Yeah, I thought it was unlikely too.  Unless it was created elsewhere, though, there's nothing *Vorta* related in the entire `.config` directory... weird.  It's OK, though.  I redid everything, which took a while, but is now functional.  Thanks for your response.
